internal/config/config.go

```diff
@@ -860,6 +860,7 @@ func fetchConfig(u *url.URL) ([]byte, error) {
 		req.Header.Add("Authorization", "Token "+v)
 	}
 	req.Header.Add("Accept", "application/toml")
+	req.Header.Set("User-Agent", "Telegraf/"+internal.Version()+" ("+runtime.GOOS+" "+runtime.GOARCH+")")
```

Comment:
Use `internal.ProductToken()` for the value, it will look something like: `Telegraf/1.14.0 Go/1.13.8` which is a rfc2616 [product token](https://tools.ietf.org/html/rfc2616#section-3.8).

Reply:
My commit based on the User-Agent Header from the http output plugin.
You mean like this.
`req.Header.Set("User-Agent", internal.ProductToken()+" ("+runtime.GOOS+" "+runtime.GOARCH+")")`

Reply:
We'll just want the product token part, I'll update the http output too
```
req.Header.Set("User-Agent", internal.ProductToken())
```